### ISSUE TITLE
Goerli network - Multisig Guild transfer proposal failed 

### ISSUE THREAD

**Issue by Filipv95** [DXgovernance/DAVI#271]
**Description**
Created Proposal with 'Transfer' action on Goerli network, voted for Proposal to pass, executed Proposal when it became 'Executable' and Proposal changed state to 'Failed'

**Steps to Reproduce**
1. SpinUp Localhost env
2. Open localhost:3000
3. Check if Landing Page is loaded
4. Connect wallet to Metamask
5. Switch to 'Goerli' network
6. Go to Multisig Guild page
7. Create Proposal with 'Transfer' action
8. Vote for Proposal to pass
9. Execute Proposal when it changes state to 'Executable'

**Expected behavior**
Proposal should have 'Executed' state and 1 SWPR token should have been transfered to selected Account addres 

**Screenshots**
![failedproposal](https://user-images.githubusercontent.com/37701286/187857153-d3f41e49-ac49-4566-82b2-a3ff2666b64c.png)

**Device Information**
OS: MacOs
Browser: GoogleChrome, Brave
Device: MacOS Monterey v12.4

**Additionaly information**
Bad description and 'Option 1' instead of 'For' is mentioned in another issue (#269)

**Comment by rossneilson** [comment]
Could you check the timeForExecution variable here, perhaps it is too short?

**Comment by Filipv95** [comment]
@rossneilson `timeForExecution` was set to 2 min and I've executed Proposal as soon as it changed state to 'Executable'.

But I've checked this now, all assets have 0 value, I think this is the reason Proposal changed state to 'Failed' but I overlooked this when I logged the issue.
![assets](https://user-images.githubusercontent.com/37701286/189313807-6c692219-6c58-4cbb-81ee-2d7f1724770f.png)

**Comment by rossneilson** [comment]
Hmmmm, this could be the case but the failed flag should be reserved for going beyond execution time limit, maybe we are handling state strangely

**Comment by Filipv95** [comment]
@rossneilson I've re-tested this. I've noticed that I was not able to execute the Proposal when it became 'Executable' until `timeForExecution` period expired - i.e. as soon as I click on 'Execute' button a 'transaction rejected' modal pops.
After 2 min expired (`timeForExecution` period) I've executed the Proposal but it went to 'Failed' state

![failedprop](https://user-images.githubusercontent.com/37701286/189326529-97a31d72-b360-4ffd-9edf-63e82ecdfef5.png)
